pkg/install/install.go

```diff
@@ -38,14 +38,23 @@ func Run(automatic bool, configFile string, powerOff bool, silent bool) error {
 		cfg.RancherOS.Install.PowerOff = true
 	}
 
-	if automatic && !cfg.RancherOS.Install.Automatic {
-		return nil
+	if reboot {
+		cfg.RancherOS.Install.Reboot = true
 	}
 
 	if silent {
 		cfg.RancherOS.Install.Automatic = true
 	}
 
+	// If we set the installation to automatic, reboot is set to true
+	if automatic {
+		cfg.RancherOS.Install.Reboot = true
```

Comment:
mmm no way to set reboot to off if automatic is set?

Reply:
well..not with the current config load and such. I mean, the flag is gonna be set to false always by default, that is the nature of the bool flags so hwo to discern between the default value and the explicit value set by an user (you can set --reboot=false/true)

I guess we could introduce a `--no-reboot` flag but goddammit, I rather redo the installer in rust  than add more corner cases.

I guess the problem here is a lack of vision/feature set well defined. Like yeah, config file should have more priority, flag should have more priority, with automatic is automatic and you cant change it, etc...


I could also use the question stuff to ask for a explicit question of reboot instead, that works both on automatic and manual install I think, but again the lack of a well defined set of features brought us here and should not guide us further expanding this as to not pile up on tech debt.

Reply:
I know that this code is really hairy to say at least, and needs a total refactor... but on the other hand by applying the patch as-is boils down to the fact that then reboot is forced and can't be changed, maybe indeed a NoReboot is better, as reboot is implicitly true by default

Reply:
>  reboot is forced and can't be changed,

Well, thats automatic install for you :P